modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/Table.js

```diff
@@ -265,6 +266,10 @@ const RowWithActions = ({
 					columnName="item-selector"
 				>
 					<SelectionComponent
+						aria-label={sub(
+							Liferay.Language.get('select-x'),
+							getLocalizableLabel(item.name, item.urls)
```

Comment:
Unfortunately, we can't make assumptions about `item` structure. Note how we deal with the item contents through the  `getValueDetailsFromItem()`  API to extract relevant values from an arbitrary item according to the field mappings provided to the component. This is basically telling us that whatever info we want to extract from the item, it should be mediated by that function, fed by the current mappings.

So, it makes no sense to attempt `item.name` from the component as it does not know what is inside the item. Just happened that the data source we use in the example (which returns Products) happens to include a `name` property which is localizable.

As FDS can't make assumptions about the contents of  `item`, all it can do to provide an `aria-label` is to guess which of the existing fields will better represent the item itself. But even this is hard because there's no clue about it. 

This is going to require an extra API property to tell FDS which of the field mappings represents the "title" of the item, so that we can use it as a representation of the item in aria labels, tooltips and such.

So, imagine we send this to the FDS (note the "title" property, which I added for illustration purposes)
```
         "schema":{
            "fields":[
               {
                  "localizeLabel":true,
                  "expand":false,
                  "fieldName":"thumbnail",
                  "contentRenderer":"image",
                  "sortable":false,
                  "label":""
               },
               {
                  "localizeLabel":true,
                  "expand":false,
                  "fieldName":[
                     "name",
                     "LANG"
                  ],
                  "contentRenderer":"actionLink",
                  "sortable":true,
                  "label":"Name"
                  "title":true
               }
```

This way, when the `aria-label` has to be computed, one can do:

```
function getAriaLabel(item, fields) {
    titleField = getTitleField(fields) ;   // this return the field which title property equals true
    return  getValueDetailsFromItem(item, titleField.fieldName)
}
```

What do you think @markocikos ?